tests/helics/core/TimeCoordinatorTests.cpp

```diff
@@ -138,3 +138,27 @@ TEST(timeCoord_tests, dependent_test_message)
     EXPECT_EQ(deps.size(), 1U);
     EXPECT_TRUE(deps[0] == fed3);
 }
+
```

Comment:
I was trying something here.  I might keep playing with it, but I figured out the issue before needing the testing here,  but left it in as I am fairly confident there will be more complex issues needing to test the specifics of the time coordinator so will need to build this out more thoroughly at some point